src/main/java/jeryl/fyp/model/FypManager.java

```diff
@@ -20,6 +20,7 @@ public class FypManager implements ReadOnlyFypManager {
 
     private final UniqueStudentList students;
 
+
```

Comment:
Any reason for this additional newline?

Reply:
Nope I have removed the additional newline